packages/expo/package.json

```
@@ -87,6 +88,7 @@
   "peerDependencies": {
     "@clerk/expo-passkeys": ">=0.0.6",
     "expo-auth-session": ">=5",
+    "expo-linking": ">=7",
```

We need to be careful with this, I believe any other `expo-*` package we have as peer is compatible with >= Expo 50 and `expo-linking@7` seems too "new". 

Link.openUrl() existed for [Expo 50](https://docs.expo.dev/versions/v50.0.0/sdk/linking/), we just need to find the correct major. Based on [their changelog](https://github.com/expo/expo/blob/sdk-50/packages/expo-linking/CHANGELOG.md) for expo 50, we can do 
```suggestion
    "expo-linking": ">=6.2.2",
```

---

Great catch on this, thanks for the heads up!